posts/2023-12-31-foss-engines-overview.md

```diff
@@ -0,0 +1,112 @@
+
+As established with Bevy’s use of ECS, the engine’s developers care deeply about modularity. The engine’s plugin system accentuates their commitment to that principle in every part of Bevy. Developers can organize the game’s systems into discrete plugins. An example of this is organizing all the UI systems code into a UI plugin. From there, developers can slot the plugin into the game’s initialization step. This helps organize the code, while also encouraging modularity by allowing the developer to add or remove the plugin based on their needs. This paradigm even applies to the engine’s core features, as they are all organized into plugins. It becomes trivial to activate or deactivate any part of the engine—including rendering, audio playback, and event loops—as the developer sees fit.
+
+Asset libraries provide a wealth of resources that empower developers to learn the tools quickly and get their game to a playable state. The community assembled a library of assets available on the official website for the community to use and share. The library includes tutorials, plugins, and templates that address subjects like physics, networking, and input management. There are even entire games and applications available in the asset library to either build on or use as a reference while learning the engine. Bevy’s structure encourages developers to use any of the resources from this library freely as part of the building blocks that will make up their game. In conjunction with Rust’s package manager, there is a strong emphasis on modularity at every level of the engine. 
+
+
+### Godot
+* Written in C++, Supported Languages: GDScript, C#
+* Platforms:
+    * Development: Windows, Linux, Mac
+    * Target: Windows, Linux, Mac, iOS, Android, WASM, HTML5
+
+Among all the modern FOSS game engines available to date, Godot has the largest and most active community. As the drama around Unity has unfolded, you have more than likely heard mentions of Godot on more than a few occasions. It is not without merit, as Godot encourages developers to shape the engine around their needs. Coming in at only 40 MB, the engine includes a lightweight, multi-platform editor capable of running on any major operating system (Windows, Mac, and Linux). In fact, you can even use a web-based version, albeit with some constraints. Godot can meet developers on whatever platform works best for them.
+
```

Comment:
Update `In fact, you can even use a web-based version`

To `In fact, you can even use a web-based and an Android-based version`